### PR TITLE
Fixed track going into rock.

### PR DESCRIPTION
**Pull request by ludvigarlebrink**

### CODE REVIEW

File: Game/Resources/Opening Scene/Opening Scene.json [hunk 1]
```diff
@@ -1384,6 +1384,117 @@
 			"scene" : false,
 			"static" : false,
 			"uid" : 1513071370
+		},
+		{
+			"Material" : 
+			{
+				"albedo" : "Resources/NewAssets/Textures/Rock/rock_albedo",
+				"metallic" : "",
```
Comment (andens) [inline Game/Resources/Opening Scene/Opening Scene.json]: Missing texture here and for roughness. This has occurred in multiple PRs by various authors. Is it a bug with saving material in the editor?

File: Game/Resources/Opening Scene/Opening Scene.json [hunk 2]
```diff
@@ -1384,6 +1384,117 @@
...
+			},
+			"scene" : false,
+			"static" : false,
+			"uid" : 1513078541
+		},
+		{
+			"Material" : 
+			{
+				"albedo" : "Resources/NewAssets/Textures/Rock/rock_albedo",
+				"metallic" : "",
```
Comment (andens) [inline Game/Resources/Opening Scene/Opening Scene.json]: Missing texture here and for roughness.

File: Game/Resources/Opening Scene/Opening Scene.json [hunk 3]
```diff
@@ -1384,6 +1384,117 @@
...
+			},
+			"scene" : false,
+			"static" : false,
+			"uid" : 1513078543
+		},
+		{
+			"Material" : 
+			{
+				"albedo" : "Resources/NewAssets/Textures/Rock/rock_albedo",
+				"metallic" : "",
```
Comment (andens) [inline Game/Resources/Opening Scene/Opening Scene.json]: Missing texture here and for roughness.

File: Game/Resources/Opening Scene/Opening Scene.json (hunk 2 above)
Comment (ludvigarlebrink) [reply Game/Resources/Opening Scene/Opening Scene.json]: Using default texture.

File: Game/Resources/Opening Scene/Opening Scene.json (hunk 3 above)
Comment (ludvigarlebrink) [reply Game/Resources/Opening Scene/Opening Scene.json]: Using default texture.

File: Game/Resources/Opening Scene/Opening Scene.json (hunk 1 above)
Comment (ludvigarlebrink) [reply Game/Resources/Opening Scene/Opening Scene.json]: Using default texture.

**Comment by Ittis** [comment]
iamheman

**Pull request merged**